queries/places.jinja2

```diff
@@ -92,6 +92,7 @@ FROM wof_neighbourhood wof_n
 INNER JOIN wof_neighbourhood_placetype wof_np ON wof_n.placetype = wof_np.placetype_code
 
 WHERE
+  wof_n.is_visible AND
```

Comment:
We don't need to - `is_visible` isn't one of the `SELECT`ed columns, so doesn't get promoted to a property in the tile. It's not present outside of the database.